src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -187,28 +217,30 @@ private Delegate MakeGetter<T>(IRow input, ColumnType columnType)
                         runner.AddInput(inputName, srcTensorGetters[i].GetTensor());
                     }
 
-                    var tensors = runner.Fetch(_outputColName).Run();
-
+                    var tensors = runner.Fetch(_outputColNames).Run();
```

Comment:
>_outputColNames [](start = 47, length = 15)

Does it make sense to create an array that only has the active outputs so that we don't need to compute and cache all of them if only some are active?

Reply:
This is basically an array passed by the user. I think any output column requested by the user must be active. Do you think a scenario where user specified column wont be active?

---
In reply to: [215390927](https://github.com/dotnet/machinelearning/pull/814#discussion_r215390927) [](ancestors = 215390927)